kili/queries/label/__init__.py

```diff
@@ -44,7 +44,7 @@ def labels(self,
                created_at_gte: Optional[str] = None,
                created_at_lte: Optional[str] = None,
                fields: List[str] = ['author.email', 'author.id', 'id',
-                                    'jsonResponse', 'labelType', 'secondsToLabel', 'skipped'],
```

Comment:
removing it from default fields can be breaking. The purpose to show deprecation warning is not warn clients before to make it breaking. We should wait until 31/08/2022 to remove it here.

Reply:
Oh, okay, thank you. Added it back